_redirects/guides/kubernetes.md

```diff
@@ -0,0 +1,4 @@
+---
+permalink: /guides/kubernetes/index.html
```

Comment:
Is this link correct?

Reply:
Yeah I know it's weird but that's how it should be. It took a lot of experiments to find the right combination :).

You can test it by going to https://quarkusio-quarkusio-github-io-preview-pr-827.surge.sh/guides/kubernetes , you should be redirected to the right quarkus.io guide.